client/src/app/site/pages/meetings/pages/motions/pages/motion-detail/components/motion-meta-data/motion-meta-data.component.ts

```diff
@@ -353,8 +353,12 @@ export class MotionMetaDataComponent extends BaseMotionDetailChildComponent impl
                 .getViewModelObservable(this.motion.id)
                 .pipe(
                     map(motion => [
-                        motion.referenced_in_motion_recommendation_extensions,
-                        motion.recommendation_extension_references as ViewMotion[]
+                        motion.referenced_in_motion_recommendation_extensions
+                            ? motion.referenced_in_motion_recommendation_extensions
+                            : null,
+                        motion.recommendation_extension_references
+                            ? (motion.recommendation_extension_references as ViewMotion[])
+                            : null
```

Comment:
I don't see how this should fix the issue. 

Trying to read a property of `null` will still result in throwing a type error. The error message in the issue looks to me like `motion` in line 355 is null. Which also does not really make sense. 

If it is an empty `motion` attribute the solution should look like this:
```suggestion
                        motion?.referenced_in_motion_recommendation_extensions,
                        motion?.recommendation_extension_references as ViewMotion[]
```